src/SeleniumLibrary/__init__.py

```diff
@@ -363,6 +363,15 @@ def run_keyword(self, name, args, kwargs):
             raise
 
     def register_driver(self, driver, alias):
+        """Add's a `driver` to the library WebDriverCache.
+
+        :param driver: Instance of the Selenium `WebDriver`.
+        :param alias: Alias given for this `WebDriver` instance.
+        :type driver: selenium.webdriver.remote.webdriver.WebDriver
+        :type alias: intger or string
```

Comment:
- I'd prefer having `:type` right after the matching `:param`. 
- Type of alias seems wrong in many ways. Should it be a class, not free text? There's a typo. And alias actually is always a string.
- Do we need to specify the type for all arguments, including strings? I noticed you hadn't added types to such arguments in other places and I think that's OK.